### ISSUE TITLE
After using this mod, I am unable to use replay

### ISSUE THREAD

**Issue by PathdarX** [TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs#59]
After using this mod, the chunks may not be fully loaded during replay recording and viewing, only a few scattered chunks can be displayed. After deleting this mod, this problem is resolved。
This is the log when i use replay: https://mclo.gs/SJbC4T3

**Comment by PathdarX** [comment]
yes，the world is in REID/JEID format，I have tested many times, all of which are replays that only show a few chunks

**Comment by PathdarX** [comment]
Sorry, I didn't find debug.log in the logs folder
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/facfd056-17e9-4e22-9bda-31192fd555f2)

**Comment by jchung01** [comment]
Then can you run with the java arg `-Dforge.logging.console.level=debug` and post the latest.log after trying to view the replay?

**Comment by PathdarX** [comment]
Sorry，When i used tihs java arg，I still couldn't find debug.log and didn't see any information output from the console，but i can post you the latest.log
https://mclo.gs/oquRJSEhttps://mclo.gs/oquRJSE

**Comment by jchung01** [comment]
Hmm, then can you run with the java arg `-Dmixin.debug.export=true`, open replay, and then zip and upload the `.mixin.out/` folder? The `.mixin.out` folder should be in the pack instance's folder (the same folder that has the `logs/` folder).

**Comment by PathdarX** [comment]
Is this the file？
[mixin.zip](https://github.com/user-attachments/files/15892088/mixin.zip)

**Comment by jchung01** [comment]
Yes, it looks like REID's mixins are being loaded correctly. I am unable to reproduce the replay issue with REID + Replay mod (and even with Biomes O Plenty), so I would need a minimal list of mods and reproduction steps to investigate further.

**Comment by PathdarX** [comment]
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/d58cc50c-52c6-4287-836d-459ea76e56de)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/1f915dcc-61dd-42c2-bd14-2261e4276e6d)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/667b8b43-1b19-409e-8de2-e02d17510b39)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/f0fda0c0-49a7-433d-8cce-2e2bc6d3eca0)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/d8cff71e-f882-4229-a888-b9220fd44f5b)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/35595882-247c-4fcf-bcd5-b6cd346ab3d4)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/2e043be8-37c9-4eed-940e-ee53bcf5adc1)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/1fe43429-1fc2-4bd0-a455-5b679caf29c3)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/86234fc4-fe66-4ffd-934a-132d010f6d0a)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/c9608d29-b00d-4824-8c18-c4914379b782)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/6c6575df-0bf7-42d0-aed4-a016b024bf5f)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/9b6fb7b4-0eed-4620-94cb-be67f232fc13)
![image](https://github.com/TerraFirmaCraft-The-Final-Frontier/RoughlyEnoughIDs/assets/133863239/198fff3c-d80b-4aca-93a2-4871819c4695)
This is my all mods

**Comment by jchung01** [comment]
Looking at this again, I believe this has to be an issue of loading an older replay that was not saved in REID format. Are you sure that this issue happens if you make a new recording, from a new world, that is created with REID, and without NEID in your pack? If your recording was saved without the presence of REID while you were recording in the real world, it would make sense that the playback recording would not be viewable correctly upon adding REID.
